app/src/main/assets/oss.txt

```diff
@@ -13,12 +13,6 @@ We are deeply grateful to these developers for their work and contributions.
 	https://spring.io
 	Copyright (c) 2013-2014 Pivotal Software, Inc
 
-* Eclipse Java compiler
-
-	https://www.eclipse.org
-	Eclipse Public License Version 1.0
-	No changes to v3.13.0
```

Comment:
Please include your changes in pull request's description.

Reply:
The PR is still WIP, I'll update description when its complete